.github/workflows/docs.yml

```diff
@@ -42,7 +42,9 @@ jobs:
       - uses: actions/checkout@v3
       - name: Setup Python
         uses: actions/setup-python@v4
+      - name: Set up venv
+        run: python -m venv .pyenv
       - name: Install tox
-        run: python -m pip install tox
-      - name: Run
-        run: tox -e docs
+        run: .pyenv/bin/python -m pip install tox
+      - name: Run tox
+        run: .pyenv/bin/python -m tox -e docs
```

Comment:
OK so I'm back to looking at this after far too long, and to refresh my memory... if we didn't use a venv here, would we get the error about `--break-system-packages`? I mean, I'd prefer not to make this change without a reason, but if using the system Python would fail unless we use `--break-system-packages`, that's enough of a reason for me. :+1:

Reply:
I think that was the case… it's been a while so my memory is fuzzy as well.

I could use `--break-system-packages`, as this is a container, a virtual environment in its own right, so nothing would break if we did use that flag (or tell `pip` to assume `--break-system-packages`), but I'm trying to break myself out of that habit. :-)

Reply:
Yeah that sounds like exactly what I was thinking. So we can just go ahead with this as is. I'll leave it to you to merge, in case there's anything else I missed.